### PR TITLE
Update mdx.py

### PR DESCRIPTION
**Pull request by mrvanes**
This fixes an unescaped user input vulnerability

### CODE REVIEW

File: src/pyff/mdx.py [hunk 1]
```diff
@@ -71,6 +71,7 @@
 from .i18n import language
 from . import samlmd
 import six
+from cgi import escape
```
[the remark this answers is not in the document]
Comment (leifj) [reply src/pyff/mdx.py]: No pyFF is 3.x and 2.x - all builds are built on 2.7, 3.5, 3.6, 3.7

[the remark this answers is not in the document]
Comment (mrvanes) [reply src/pyff/mdx.py]: Maybe in master now, but our deploy is still a 2.7 venv. I expect many others are, due to legacy. Do you want to break all those installations without warning?

[the remark this answers is not in the document]
Comment (leifj) [reply src/pyff/mdx.py]: Yeah sorry. I took your comment to mean that pyFF didn't do 3.x but you meant that pyFF still needs to maintain backwards compat with 2.x.

[the remark this answers is not in the document]
Comment (leifj) [reply src/pyff/mdx.py]: Can't we simply do a if six.PY2: ... etc?

[the remark this answers is not in the document]
Comment (mrvanes) [reply src/pyff/mdx.py]: I don't know six-fu, please consider this PR a best-effort favor to the community...

[the remark this answers is not in the document]
Comment (c00kiemon5ter) [reply src/pyff/mdx.py]:
```py
import six
if six.PY2:
    from cgi import escape
else:
    from html import escape
```

[the remark this answers is not in the document]
Comment (leifj) [reply src/pyff/mdx.py]: I'm fine merging this and fixing the six move myself

**Comment by leifj** [comment]
Kudos to Tirtha Mandal <mandaltirtha17@gmail.com> and Jonas Lejon <jonas@triop.se> who independently discovered this. Closes #160

**Pull request merged**